pkg/apis/troubleshoot/v1beta1/collector_types.go

```diff
@@ -20,6 +20,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// CollectorSpec defines the desired state of Collector
+type CollectorSpec struct {
+	Collectors      []*Collect `json:"collectors,omitempty" yaml:"collectors,omitempty"`
+	UploadResultsTo string     `json:"uploadResultsTo,omitempty" yaml:"uploadResultsTo,omitempty"`
```

Comment:
Could these be put into a single struct? Like postCollectionActions?  Which could be an array? Since there are multiple items here, it's not obvious to me how to use these